mailpoet/tests/performance/tests/forms-subscribing.js

```diff
@@ -0,0 +1,106 @@
+/**
+ * External dependencies
+ */
+import { sleep } from 'k6';
+import { browser } from 'k6/experimental/browser';
+import { randomIntBetween } from 'https://jslib.k6.io/k6-utils/1.1.0/index.js';
+import {
+  expect,
+  describe,
+} from 'https://jslib.k6.io/k6chaijs/4.3.4.2/index.js';
+
+/**
+ * Internal dependencies
+ */
+import {
+  baseURL,
+  thinkTimeMin,
+  thinkTimeMax,
+  formsPageTitle,
+  fullPageSet,
+  screenshotPath,
+} from '../config.js';
+import {
+  login,
+  waitAndType,
+  waitForSelectorToBeVisible,
+} from '../utils/helpers.js';
+
+export async function formsSubscribing() {
+  const page = browser.newPage();
+
+  try {
+    let subscriberEmail = 'blackhole+testform' + Date.now() + '@mailpoet.com';
+
```

Comment:
I assume we are using the MailPoet sending service? 
The `blackhole` address works only with it.

Reply:
Yes, this test requires MSS so we will use blackhole ones.